carbon-design-system/carbon
# Add storybook v6 support to `@carbon/react`

Closes #8554

This PR adds in the initial version of Storybook v6 to the `@carbon/react`. However, there are some known issues/limitations that we ran into that will be addressed in follow-up PRs:

- Issues
  - Welcome page styling issues
  - No base styles applied to custom MDX docs
  - Performance problems when importing from the `carbon-components-react` entrypoint
- Enhancements
  - Theme switcher using built-in Storybook functionality 
  - Locale switcher using built-in Storybook functionality
  - Deployments through Netlify
  - Add-ons panel 

#### Changelog

**New**

- Add storybook v6 support to `@carbon/react`

**Changed**

**Removed**

#### Testing / Reviewing

- `cd packages/carbon-react`
- Run `yarn storybook`
- Verify that the project runs and that you can visit the Grid story
- Verify that the reset styles are applied from the sass imports (it should use Plex instead of the default sans font for your OS)
